Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -241,7 +242,24 @@ bool ES::LaunchTitle(u64 title_id, bool skip_reload)
   // (supposedly when trying to re-open those files).
   DiscIO::NANDContentManager::Access().ClearCache();
 
-  if (IsTitleType(title_id, IOS::ES::TitleType::System) && title_id != TITLEID_SYSMENU)
+  u32 device_id;
+  constexpr u32 DEFAULT_DEVICE_ID = 0x0403ac68;
```

Comment:
I don't think there's much point to pointing out in which other locations this constant is used.